src/onegov/org/locale/de_CH/LC_MESSAGES/onegov.org.po

```diff
@@ -3418,6 +3421,15 @@ msgstr "Anmeldung"
 msgid "The ticket number is"
 msgstr "Ihre Ticket Referenz"
 
+msgid "The following reservations have been rejected:"
+msgstr ""
+
+msgid ""
+"This is a notification for the rejected reservations for ${organisation}. If "
+"you no longer wish to receive these notifications, please contact an "
+"administrator so they can remove you from the recipients list."
+msgstr ""
```

Comment:
It's possible I accidentally removed these when merging master. These translations seem to exist for the other languages.